test/mlir/conversion/onnx_to_tosa/Math/Elementwise.mlir

```diff
@@ -28,10 +30,49 @@ func.func @test_neg(%arg0: tensor<10x10xf32>) -> tensor<10x10xf32> {
 // CHECK-NEXT:      [[VAR_0_:%.+]] = "tosa.negate"([[PARAM_0_]]) : (tensor<10x10xf32>) -> tensor<10x10xf32>
 }
 
+// -----
```

Comment:
I suppose that we should have an empty line before and after `// -----` to make it work.